src/pages/patterns/read-only-states-pattern/index.mdx

```diff
@@ -0,0 +1,360 @@
+  modify and remove a component’s interactive functions.
+---
+
+<PageDescription>
+
+Read-only states are applied to components that users can review but not modify
+and remove a component’s interactive functions.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Overview</AnchorLink>
+  <AnchorLink>Formatting</AnchorLink>
+  <AnchorLink>Visual guidance</AnchorLink>
+  <AnchorLink>Content</AnchorLink>
+  <AnchorLink>Accessibility</AnchorLink>
+  <AnchorLink>Related</AnchorLink>
+  <AnchorLink>References</AnchorLink>
+  <AnchorLink>Feedback</AnchorLink>
+</AnchorLinks>
+
+## Overview
+
+Read-only states are applied to components that can be modified in their enabled
+state. These components are considered active, and the data they contain can be
+used in an application’s processes. This state change transforms a component’s
+purpose to be purely informative.
+
+<Row>
+<Column colLg={12}>
+
+![Form component read-only states in context.](/images/read-only-full-image.png)
+
+<Caption>Form component read-only states in context.</Caption>
+
+</Column>
+</Row>
+
+### When to use
+
+There are three main use cases that initiate a read-only state:
+
+| Use case            | Description                                                                                                     |
+| ------------------- | --------------------------------------------------------------------------------------------------------------- |
+| Application process | An application’s process temporarily restricts a user from modifying the component until the process completes. |
+| Locked              | An application restricts the number of users that can modify the component at the same time.                    |
+| Permissions         | A user’s credentials allow them to view the component but not modify it.                                        |
+
+### When not to use
+
+- If the component does not have an enabled state, do not use a read-only state
+  to display static information.
+- As an alternative to a disabled state, read-only and disabled states serve
+  different purposes. For example, when a component is temporarily unavailable
+  pending user actions or decisions (such as completing a form or choosing an
+  option), the component's state should be temporarily disabled, not read-only.
+- If the component would otherwise be disabled, components in a disabled state
+  should not change to a read-only state.
+
+## Formatting
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Read-only state anatomy.](/images/read-only-anatomy.png)
+
+<Caption>Read-only state anatomy.</Caption>
+
+</Column>
+</Row>
+
+1. **Background color change:** Having a transparent background for fields.
+2. **Border color change:** De-emphasizing selection and clickability to make
+   information more readable.
+3. **Icon color change:** Keepingg embedded icons in the component for context
+   but displaying that icons are not interactive with color and cursor changes.
```

Comment:
I think we should add a 4th around the type
"Text color no change: text color remains the same as in the enabled state and should still pass 4.5:1 color contrast rules."

Reply:
Good idea! Will add 👍